Demo/UI Tests/PayPal UI Tests/PayPal_BillingAgreement_UITests.swift

```diff
@@ -48,7 +48,7 @@ class PayPal_BillingAgreement_UITests: XCTestCase {
     func testPayPal_billingAgreement_cancelsSuccessfully_whenTappingCancelButtonOnPayPalSite() {
         let webviewElementsQuery = app.webViews.element.otherElements
 
-        self.waitForElementToAppear(webviewElementsQuery.links["Cancel Sandbox Purchase"])
+        self.waitForElementToAppear(webviewElementsQuery.links["Cancel Sandbox Purchase"], timeout: 20)
```

Comment:
This change isn't directly related to this PR, but I saw the test failing in CI, so I increased the timeout to get CI passing. (Same thing on the file below.)

Reply:
🙏